frontends/web/src/routes/account/send/services.ts

```diff
@@ -37,8 +37,7 @@ export const txProposalErrorHandling = (registerEvents: () => void, unregisterEv
     return { feeError: t(`send.error.${errorCode}`) };
   default:
     if (errorCode) {
-      unregisterEvents();
-      alertUser(errorCode, { callback: registerEvents });
+      alertUser(errorCode);
```

Comment:
this has the side-effect that if an unknown error occurred and Alert is shown the user can hit the esc key and go back.